### PR TITLE
OKTA-541264 - Release note entry for 2022.10.2

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added classic and oie release notes for this weekly release.
- **Is this PR related to a Monolith release?** 2022.10.2

### Resolves:

* [OKTA-541264](https://oktainc.atlassian.net/browse/OKTA-541264)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,33 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+#### Bugs fixed in 2022.10.2
+
+* A `/token` request that used the `code` parameter failed when the **Groups claim type** in the app was set to **Expression** and the group number exceeded 100. (OKTA-518536)
+
+* A `/token` request that used the `code` parameter and the `refresh_token` grant type failed when the **Groups claim type** in the app was set to either **Expression** or **Filter** and the group number exceeded 100. (OKTA-531605)
+
+* When a user entered their credentials in the widget (version 6.6.1) during self-service registration, their information was cleared if the username didn't exist in the org. (OKTA-532293)
+
+* The MyAccount API sent no notifications for Add Email calls with a false `sendEmail` parameter. (OKTA-539268)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:28]:
```suggestion
* The MyAccount API didn't sent notifications for Add Email calls with a false `sendEmail` parameter. (OKTA-539268)
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 2]
```diff
@@ -6,6 +6,33 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+* A user with a current session in Classic Engine couldn't be deleted (`DELETE /session/me`) after upgrading to Identity Engine. (OKTA-517093)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:30]:
```suggestion
* An opened session in Classic Engine couldn't be deleted (`DELETE /session/me`) after upgrading to Identity Engine. (OKTA-517093)
```
I know this was already approved, so you can ignore this suggestion. The original sentence sounds like you are trying to delete the user (not the session). I interpret "with a current session in Classic Engine" as a modifier.

**Review by vanngo-okta — APPROVED** [review]
Small nit and suggestion. Approving to not hold you up. 👍

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 3]
```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
...
+
+#### Bugs fixed in 2022.10.2
+
+* A `/token` request that used the `code` parameter failed when the **Groups claim type** in the app was set to **Expression** and the group number exceeded 100. (OKTA-518536)
+
+* A `/token` request that used the `code` parameter and the `refresh_token` grant type failed when the **Groups claim type** in the app was set to either **Expression** or **Filter** and the group number exceeded 100. (OKTA-531605)
+
+* When a user entered their credentials in the widget (version 6.6.1) during self-service registration, their information was cleared if the username didn't exist in the org. (OKTA-532293)
+
+* When a user authenticated using the device authorization flow, the user had to visit the verification URI twice when agentless desktop single sign-on (DSSO) was configured. (OKTA-501601)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:21]:
```suggestion
* When a user authenticated using the device authorization flow, the user had to visit the verification URI twice when Agentless Desktop Single Sign-On(DSSO) was configured. (OKTA-501601)
```
Agentless DSSO was in the Capitalization guidelines

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:30]: Great catch. I wanted to change this one, too. I'll pass by Graham for another look for all of these.

**Pull request merged**